src/lwip/standalone/sys_arch.c

```diff
@@ -687,7 +687,7 @@ void ppp_trace(int level, const char * format, ...)
 }
 #endif
 
-#if LWIP_DEBUG
+#ifdef LWIP_DEBUG
```

Comment:
nothing sets it to 0 if undef?

Reply:
Inside lwip, the test is

`#ifdef LWIP_DEBUG`

Therefore 0 == true

Reply:
thx for bringing me up to speed